pkg/apis/scheduling/v1alpha2/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2019 and  Volcano Authors

Reply:
nop, this is a cherry-pick, we need to keep this copyright.

Reply:
ic